.github/workflows/check.yml

```diff
@@ -0,0 +1,27 @@
+name: Build debug APK
```

Comment:
Shouldn't this be changed to "Prettify Kotlin Code", "Check Kotlin Style" or something similar?

Also I think having a more descriptive file name (like `check_kotlin_style`) could be better too.

Reply:
Ohhhh, I copied from `build_debug_apk.yml`, missed it.

Reply:
f782a64c84270ccdf94525aed0aa23bfb381c2ce